### PR TITLE
Ryan's styling updates

### PR DESCRIPTION
**Pull request by skoolbus39**
I'll probably have some more minor refinements, but this is getting pretty close.

### CODE REVIEW

File: www/templates/default/html/Controller.tpl.php [hunk 1]
```diff
@@ -111,33 +111,41 @@
...
-                            <div class="wdn-input-group">
-                                <input type="text" name="q" id="searchinput" title="Search Query" placeholder="e.g., Monday, tomorrow" value="<?php if (isset($context->options['q'])) { echo $context->options['q']; } ?>" />
-                                <span class="wdn-input-group-btn">
-                                    <button type="submit" class="wdn-icon-search" title="Search"></button>
-                                </span>
-                            </div>
-                        </form>
-                    </div>
-                </div></div>
+                <div class="wdn-inner-wrapper">
```
Comment (mfairchild365) [inline www/templates/default/html/Controller.tpl.php:114]: Indentation and white space is all messed up in this file.  Its okay for now, but it helps with readability and identifying changes during the code review if changes like this are not committed.

File: www/templates/default/html/EventInstance/Date.tpl.php [hunk 2]
```diff
@@ -14,15 +14,14 @@
...
     <?php endif; ?>
 </span>
 <span class="time-wrapper eventicon-clock">
     <?php if ($context->isAllDay()): ?>
     All Day
     <?php else: ?>
-        <?php echo $startu->format('g:i a')?><?php if (!empty($endtime) && $endtime != $starttime): ?>-<?php echo $endu->format('g:i a')?><?php endif; ?>
+        <?php echo $startu->format('g:i a')?><?php if (!empty($endtime) && $endtime != $starttime): ?>&ndash;<?php echo $endu->format('g:i a')?><?php endif; ?>
     <?php endif; ?>
-
```
Comment (mfairchild365) [inline www/templates/default/html/EventInstance/Date.tpl.php:27]: Another whitepsace change. :(

File: www/templates/default/html/js/events.js [hunk 3]
```diff
@@ -175,22 +175,21 @@ require(['jquery', 'wdn', 'modernizr'], function($, WDN, Modernizr) {
 		if ($sidebarCal.length) {
 			determineActiveDay();
 			addMonthWidgetStates();
-			stickySidebar();
```
Comment (mfairchild365) [inline www/templates/default/html/js/events.js:189]: Is the stickySidebar() call supposed to be removed?  I don't see it anywhere else.

**Comment by mfairchild365** [comment]
More merge conflicts :(

Also, I still see several whitespace problems in other files.

**Comment by kabel** [comment]
The resolved merge conflict were _not_ done well. Quite a bit of JS functionality has regressed.

**Comment by mfairchild365** [comment]
@kabel thanks for finding that.  What exactly was regressed?  Should I revert the this pull request?

**Pull request merged**